src/main/java/com/conveyal/gtfs/GTFS.java

```diff
@@ -252,7 +324,7 @@ private static Options getOptions () {
                 .desc("zip file path for the exported GTFS").build());
         options.addOption(Option.builder().longOpt("load").hasArg()
                 .argName("file").desc("load GTFS data from the given file").build());
-        options.addOption(Option.builder().longOpt("validate").hasArg().optionalArg(true).argName("feed")
+        options.addOption(Option.builder().longOpt("validate").hasArg().optionalArg(true).argName("feedId")
```

Comment:
"feedId" has another meaning distinct from our unique-IDs for loaded feed versions ("namespaces"). We should probably avoid use of the word feedId.

Reply:
I would agree with that.  However, it has been used in a number of places within this class already to mean `namespace` often with a parenthetical clarifying that it is referring to the schema identifier.  I will simply update all of the cmd line option args/text for now.

Reply:
I don't necessarily think that we should change this right away, now that I see how many times the word feedId is used in this class. But I don't see why it would be a breaking change. The validate tasks's argument is called `feed` rather than `feedId` so the command line options are already a bit incoherent. But I don't even think the argument names are used anywhere. We'd want to also change some parameter names, but since parameters are positional in Java it wouldn't break callers.

I will make a ticket for this if we're not going to change it soon.

Reply:
Oh, you're right.  Changing the name `feedId` wouldn't affect any command line instructions, just the output of the help text.  I will update these arg names here and we can use the ticket you create to track down/update other inconsistent usages.